includes/util.php

```diff
@@ -76,3 +76,454 @@ function newspack_install_activate_theme() {
 	switch_theme( $theme_slug );
 	return true;
 }
+
+/**
+ * Get full list of currency codes. Copied from https://github.com/woocommerce/woocommerce/blob/master/includes/wc-core-functions.php
+ *
+ * @return array
+ */
+function newspack_currencies() {
+	$currencies = array(
+		'AED' => __( 'United Arab Emirates dirham', 'woocommerce' ),
```

Comment:
We probably want `newspack` as the textdomain, right?

Reply:
Yeah, good catch: https://github.com/Automattic/newspack-plugin/pull/114/commits/229ca041d293900d763d9a93de3eee43e9bf4b0f